packages/server/socketHandlers/handleMessage.ts

```diff
@@ -28,9 +28,10 @@ const handleParsedMessage = async (
       return
     }
     const response = await handleGraphQLTrebuchetRequest(msg, connectionContext)
-    if (response) {
+    // only reply if an opId was included. no opId = no sink on client = ignored
+    if (response?.id) {
```

Comment:
this is basically the whole PR.
if it came in via websocket, don't reply if the client didn't give you an opId